model/task_lifecycle.go

```diff
@@ -1643,26 +1643,57 @@ func ClearAndResetStrandedHostTask(h *host.Host) error {
 	return nil
 }
 
-// ResetStaleTask fixes a task that has exceeded the heartbeat timeout by
-// marking the current task execution as finished and, if possible, a new
-// execution is created to restart the task.
+// ResetStaleTask fixes a task that has either exceeded the heartbeat timeout
```

Comment:
Can we call this function something else, like `FixStaleTask` or something, since it no longer resets?